Tests/ParselyTestCase.swift

```diff
@@ -3,15 +3,26 @@ import XCTest
 
 class ParselyTestCase: XCTestCase {
     internal var parselyTestTracker: Parsely!
-    static let testApikey: String = "examplesite.com"
 
     override func setUp() {
         super.setUp()
-        parselyTestTracker = Parsely.getInstance()
+        parselyTestTracker = makePareslyTracker()
     }
-    
-    override func tearDown() {
-        parselyTestTracker.hardShutdown()
-        super.tearDown()
+
+    func makePareslyTracker() -> Parsely {
+        let tracker = Parsely.getInstance()
+
+        // Note that because we call addTeardownBlock, this method needs to be defined within an
+        // XCTestCase
+        addTeardownBlock {
+            tracker.hardShutdown()
+        }
+
+        return tracker
     }
```

Comment:
Using `addTeardownBlock` in the context of `makeParselyTracker()` makes sure that any test calling this method will call `hardShutdown()` on completion. This gives us the ability to call the method and assign its return value to an instance scoped at the test method without the risk of it remaining in memory after the test finishes.